packages/web-components/src/components/ic-select/ic-select.tsx

```diff
@@ -850,6 +850,8 @@ export class Select {
                 <input
                   class="select-input"
                   role="combobox"
+                  type="search"
```

Comment:
Is this required when we autocomplete is set to off below?

Reply:
It unusually is, weird caveat of the input element.
I considered other solutions such as setting autoComplete="new-password" etc. But this made the most sense in the context.
I believe it also adds to the semantics since it is a search box (although annoyingly it does need that CSS to remove the default "x")

Reply:
Agreed, tested again and autocomplete="off" worked by itself.